### DISCUSSION TITLE
"Quick reaction" as review comment

### DISCUSSION THREAD

**Discussion by MrTact** [community/community#8901] [Pull Requests]
Suggestion: add an emoji option to the "+" menu that appears for each line of code. This would allow for quick reactions to code without having to write a comment, encouraging positive feedback (a quick "👍🏻" to say "well done!" or "I like this")

**Comment by fargusplumdoodle** [comment]
I would love something like this! I feel like my reviews often have to be focusing on what is wrong. It would add a lot of clutter to the PR if I responded with a bunch of comments that were just emojis. 

I feel like if this were to be implemented it would have to be subtle. Maybe just outside the code.

**Comment by coryfklein** [comment]
I was just about to post this myself as a feature request titled _Feature request: Inline Emojis In PR_. I'll just put the body of my own request here instead.

Often I want to simply leave a single emoji (often 👍  or 🎉  or ❤️  or 🙏 ) to signal a simple response to a code change in a PR. Leaving such an emoji in a full-fledged _comment_ is quite disruptive to the flow of the page though, so I usually don't leave any feedback at all.

![image](https://user-images.githubusercontent.com/3503322/195716269-c24c6d2d-55df-4a96-9f87-4e96e8a940fb.png)

But I _want_ to praise and give these kinds of small thanks! My request is: can we please have a method of leaving a simple in-line "react-moji" style emoji? Maybe something like this?

![image](https://user-images.githubusercontent.com/3503322/195716795-6d4bce2a-d2f5-4dfc-bf1f-b3356bc1adfe.png)

**Reply by fargusplumdoodle** [reply]
This is exactly what I want! Thanks for taking the time to make pictures for it

**Comment by tonglil** [comment]
Exactly! [Opened a dupe](https://github.com/orgs/community/discussions/56362) before I found this.
